src/app/credExplorer/pagerankTable/TableRow.test.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+
+import React from "react";
+import {shallow} from "enzyme";
+import {TableRow} from "./TableRow";
+
+import {COLUMNS} from "./sharedTestUtils";
+require("../../testUtil").configureEnzyme();
+
+describe("app/credExplorer/pagerankTable/TableRow", () => {
+  function example(depth: number = 3) {
+    return shallow(
+      <TableRow
+        depth={depth}
+        description={<span testDescription={true} />}
```

Comment:
Adding custom properties like this incurs a warning (which Enzyme
silences, annoyingly):

> Warning: React does not recognize the `testDescription` prop on a DOM
> element. If you intentionally want it to appear in the DOM as a custom
> attribute, spell it as lowercase `testdescription` instead. If you
> accidentally passed it from a parent component, remove it from the DOM
> element.

Personally, I’d spell it `data-test-description`:

```diff
diff --git a/src/app/credExplorer/pagerankTable/TableRow.test.js b/src/app/credExplorer/pagerankTable/TableRow.test.js
index ec29ed3..0450bf2 100644
--- a/src/app/credExplorer/pagerankTable/TableRow.test.js
+++ b/src/app/credExplorer/pagerankTable/TableRow.test.js
@@ -12,10 +12,10 @@ describe("app/credExplorer/pagerankTable/TableRow", () => {
     return shallow(
       <TableRow
         depth={depth}
-        description={<span testDescription={true} />}
+        description={<span data-test-description={true} />}
         connectionProportion={0.5}
         score={133.7}
-        children={<div testChildren={true} />}
+        children={<div data-test-children={true} />}
       />
     );
   }
@@ -58,9 +58,9 @@ describe("app/credExplorer/pagerankTable/TableRow", () => {
   it("displays children only when expanded", () => {
     const el = example();
     el.setState({expanded: false});
-    expect(el.find({testChildren: true})).toHaveLength(0);
+    expect(el.find({"data-test-children": true})).toHaveLength(0);
     el.setState({expanded: true});
-    expect(el.find({testChildren: true})).toHaveLength(1);
+    expect(el.find({"data-test-children": true})).toHaveLength(1);
   });
   it("has the correct number of columns", () => {
     const el = example();
@@ -96,6 +96,6 @@ describe("app/credExplorer/pagerankTable/TableRow", () => {
     const td = example()
       .find("td")
       .at(index);
-    expect(td.find({testDescription: true})).toHaveLength(1);
+    expect(td.find({"data-test-description": true})).toHaveLength(1);
   });
 });
```

but up to you.